### DISCUSSION TITLE
How to use math set operators

### DISCUSSION THREAD

**Discussion by rhthomas** [tonsky/FiraCode#1535] [Q&A] [answered]
I am writing TLA+ and have the following code:
```tla
NoOverdrafts == \A p \in People: acct[p] >= 0
```
However the only ligature that works is `>=`. How do I enable the math set operators? Ideally it would view as:
```tla
NoOverdrafts == ∀ p ∈ People: acct[p] ≥ 0
```

**Comment by wopian** [comment, marked as answer]
Those are not ligatures in Fira Code

**Reply by rhthomas** [reply]
Ah, perhaps I misunderstood [this](https://github.com/tonsky/FiraCode/blob/master/extras/math.png). So this means the characters are part of the font, but not ligatures?

**Reply by wopian** [reply]
Yeah, the unicode characters themselves are part of the font
